scala3doc/src/dotty/dokka/tasty/ClassLikeSupport.scala

```diff
@@ -123,6 +123,26 @@ trait ClassLikeSupport:
           .map { _ =>
             parseMethod(dd.symbol, kind = Kind.Given(getGivenInstance(dd).map(_.asSignature), None))
           }
+          
+      case dd: DefDef if !dd.symbol.isHiddenByVisibility && dd.symbol.isExported =>
+        val exportedTarget = dd.rhs.flatMap { 
+          case a: Apply => Some(a.fun)
+          case s: Select => Some(s)
+          case _ => None
+        }.map { 
+          case s: Select => s
+        } 
+        val functionName = exportedTarget.fold("instance")(_.name)
+        val instanceName = exportedTarget.fold("function")(_.qualifier.asInstanceOf[Select].name)
+        val dri = dd.rhs.flatMap {
+          case a: Apply => None
+          case s: Select => 
+            val dri = s.symbol.dri 
+            dri.getCallable match
```

Comment:
we should check if symbol is method not check if callable is set in DRI

Reply:
As I recall it was my first attempt then it came out everything he perceives as method, so I had to find another way to determine it, but I will double check that